A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -117,6 +117,18 @@ class CDUInitPage {
         Coherent.trigger("AP_VS_VAL_SET", 300);
         Coherent.trigger("AP_HDG_VAL_SET", 180);
     }
+    // Does not refresh page so that other things can be performed first as necessary
+    static async updateTowIfNeeded(mcdu) {
+        if (isFinite(mcdu.taxiFuelWeight) &&
+            isFinite(mcdu.zeroFuelWeight) &&
+            isFinite(mcdu.blockFuel)) {
+
+            const tow = mcdu.zeroFuelWeight +
```

Comment:
Should be one line ;)

Reply:
Resolved https://github.com/flybywiresim/a32nx/pull/600/commits/559844ee8f7035558291457dded1d21e84223bff.